src/services/utilities.ts

```diff
@@ -1390,11 +1390,11 @@ namespace ts {
     }
 
     export function getPossibleGenericSignatures(called: Expression, typeArgumentCount: number, checker: TypeChecker): readonly Signature[] {
-        let type = checker.getTypeAtLocation(called);
-        if (isOptionalChain(called.parent)) {
-            type = removeOptionality(type, isOptionalChainRoot(called.parent), /*isOptionalChain*/ true);

```

Comment:
I don’t immediately understand why this check has been removed—is it really unnecessary, or is it just untested?